ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/AffectedDeploymentsTable.tsx

```diff
@@ -86,6 +87,7 @@ function AffectedDeploymentsTable({
                     <Th>First discovered</Th>
                 </Tr>
             </Thead>
+            {deployments.length === 0 && <EmptyTableResults colSpan={7} />}
```

Comment:
Having to specify `colSpan` == the number of columns per table is making me lean towards a "tableColumnDescriptors" type approach for these tables, similar to how Violations is done, instead of directly adding in the headers/cells.

Probably not worth the extra bit of complexity and time to refactor at this point though, unless we have a lot of time at the end of this release. What do you think?

Reply:
i personally like the declarative columns over the iterative columns unless we're planning on dynamically adding/removing columns often because it keeps things straight forward and easy to find. idk if it's worth abstracting to avoid a `colSpan` prop.